.NET/Microsoft.Recognizers.Text.DateTime/Italian/Parsers/ItalianTimeParserConfiguration.cs

```diff
@@ -36,16 +42,16 @@ public void AdjustByPrefix(string prefix, ref int hour, ref int min, ref bool ha
             var deltaMin = 0;
             var trimmedPrefix = prefix.Trim();
 
-            // c'este 8 heures et demie, - "it's half past 8"
-            if (trimmedPrefix.EndsWith("demie"))
+            // "it's half past 8"
+            if (trimmedPrefix.EndsWith("mezza") || trimmedPrefix.EndsWith("mezzo"))
```

Comment:
Ideally these hardcoded words should be moved to YAML. But this is not the highest priority now.
Looks OK to merge once conflicts are resolved.